packages/ui/src/styles/global/css-variables.scss

```diff
@@ -50,4 +51,27 @@
   /* Sizes */
   --va-form-element-min-width: 50px;
   --va-form-element-default-width: 250px;
+
+  /* Colors */
+  // Accent
+  --va-primary: #154ec1;
```

Comment:
We've suggested not to use global variables, no?

Reply:
With `:host` it will be a global variable inside Shadow DOM, which makes each instance encapsulated. `:root` should be changed in different PR.